extensions-contrib/opentelemetry-emitter/src/test/java/org/apache/druid/emitter/opentelemetry/OpenTelemetryEmitterTest.java

```diff
@@ -70,6 +70,7 @@ public CompletableResultCode shutdown()
     }
   }
   private static final DateTime TIMESTAMP = DateTimes.of(2021, 11, 5, 1, 1);
+  private static int DEFAULT_ATTRIBUTES_SIZE = 1;
```

Comment:
It might be better to do something like this

```java
class OpenTelemetryEmitter {
  static final Map<String, String> DEFAULT_SPAN_ATTRIBUTES = ImmutableMap.of(
    "exporter.kafka", "exclude"
  );

  ...
  DEFAULT_SPAN_ATTRIBUTES.entrySet().stream().forEach((k, v) -> span.setAttribute(k, v));
}
```
```java
class OpenTelemetryEmitterTest {
  Assert.assertEquals(DEFAULT_SPAN_ATTRIBUTES.size(), actualSpanData.getAttributes().size());
}
```